manager/integration/tests/test_basic.py

```diff
@@ -1368,11 +1368,10 @@ def test_deleting_backup_volume(client, volume_name):  # NOQA
     volume.attach(hostId=lht_host_id)
     volume = common.wait_for_volume_healthy(client, volume_name)
 
-    bv, _, snap1, _ = create_backup(client, volume_name)
-    _, _, snap2, _ = create_backup(client, volume_name)
+    create_backup(client, volume_name)
+    create_backup(client, volume_name)
 
     delete_backup_volume(client, volume_name)
-    cleanup_volume(client, volume)
```

Comment:
Why `create_volume` removed? It will not impact the test result, but we should keep it for now.

Reply:
I thought this looks duplicated since it's already handled by the cleanup_client. But on another thought, it makes sense to test for volume cleanup.

Reply:
Addressed.